src/shared_utils/shared_utils.star

```diff
@@ -1,7 +1,7 @@
 TCP_PROTOCOL = "TCP"
 UDP_PROTOCOL = "UDP"
-
-
+HTTP_APPLICATION_PROTOCOL = "http"
+OPTIONAL_APPLICATION_PROTOCOL = ""
```

Comment:
I probably would have called this `NOT_PROVIDED_APPLICATON_PROTOCOL` or something like this, since the "optional" actually pertains to the applicationProtocol arg and not this particular constant